### DISCUSSION TITLE
Paperless - Bad Request (400) error

### DISCUSSION THREAD

**Discussion by Valve90210** [alexbelgium/hassio-addons#1080] [General]
I've gone to use Paperless this morning and am being greeted by a Bad Request (400) error. Looking at the add on log I can see something about a django.security.DisallowedHost message.

I've seen some of the posts about the config file location etc and I've noticed various notifications about available updates etc on HAOS but wasn't sure if my error is related to that, so thought I'd check before posting it as an issue?

**Comment by alexbelgium** [comment]
Hi sorry indeed paperless is a mess now that I upgraded from the old lsio version (1.15) to the new image (2.0). I'll try to solve that asap

To avoid this message, could can use the paperless_url (something like that) in the options to set what is your host (for example : http://myhost)

**Comment by Valve90210** [comment]
Not a problem, I just wanted to check the issues I know you're having were behind my error message. I have no urgent need to use it right now so I'll just leave it be and allow time for you to work your magic! Really appreciate all the work you do with the addons, it's added so much to Home Assistant for me personally!!

**Comment by alexbelgium** [comment]
Thanks ! I'm working on that but it is surprisingly difficult

Thanks very much for the kind feedback !
